alidock/__init__.py

```diff
@@ -182,6 +183,15 @@ def run(self):
         if platform.system() != "Linux":
             dockMounts.append(Mount("/persist", "persist-"+self.conf["dockName"], type="volume"))
 
+        if self.conf["cvmfs"]:
+            if os.path.isdir("/cvmfs"):
+                if "Linux" in platform.system():
+                    dockMounts.append(Mount("/cvmfs", "/cvmfs", type="bind", propagation="shared"))
```

Comment:
Given that `propagation` defaults to `None` (see [here](https://docker-py.readthedocs.io/en/stable/api.html#docker.types.Mount)) I'd do something like (ignore the non-PEP-8 indentation):

```py
dockMounts.append(Mount("/cvmfs", "/cvmfs", type="bind",
  propagation="shared" if platform.system() == "Linux" else None))
```

If host's `/cvmfs` does not exist and you eliminate the error check, what happens? Does `/cvmfs` get created? I am asking, because I think the API behaves differently than the `docker` command to this respect.

In case the check is needed: change the error message to:

```
/cvmfs not found, cannot mount
```

Reply:
Yes, the APIs behave differently. No need for checking.